mobile/Screen/EventDetailScreen.js

```diff
@@ -99,32 +291,117 @@ function EventDetailScreen({
     }
   };
 
+  // TO DO: add 'first' tuesdays, etc.
+  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const dayOfWeek = daysOfWeek[day];
+  const weeksOfMonth = ['First', 'Second', 'Third', 'Fourth'];
+  const weekOfMonth = weeksOfMonth[week - 1];
+
+  const saveIcon = isEventSaved ? require('../assets/detailScreen/saved.png') : require('../assets/detailScreen/unsaved.png');
+
   return (
-    <ScrollView style={styles.container}>
-      <View style={styles.heading}>
-        <Text style={styles.h1}>{title}</Text>
-        <Switch value={isEventSaved} onValueChange={onPressToggleSavedEvent} title="Event Save Button" />
-      </View>
-      <Text onPress={onPressEvent}>{organizations}</Text>
-      <View style={styles.border} />
-      <View style={styles.square} />
-      <Text>{day}</Text>
-      <View style={styles.square} />
-      <Text>{time}</Text>
-      <View style={styles.square} />
-      <Text>{location}</Text>
-      <View style={styles.border} />
-      <Text style={styles.h2}>Event Description</Text>
-      <Text>
-        {' '}
-        {summary}
-        {' '}
-      </Text>
-      <Text style={styles.h2}>Pictures</Text>
+    <ScrollView contentContainerStyle={styles.container}>
```

Comment:
Also not exactly sure what the issue is but on my iPhone on this (event details) screen,  the scrolling is not just vertical it's also horizontal? there's nowhere to actually scroll horizontally obv but when I try to scroll vertically, my screen also bounces left and right, if that makes sense. I'll send a video of what I mean. It looks like a styling error, though I also can't reproduce it on my emulator so idk